app/helpers/lti_helper.rb

```diff
@@ -1,5 +1,45 @@
 module LtiHelper
-    def lms_integration_tutorial
-        return "https://www.example.com" # link to the tutorial of linking cv assignment with LMS
+  def lms_integration_tutorial
+    return "https://www.example.com" # link to the tutorial of linking cv assignment with LMS
+  end
+
+  def lti_score_submit(lis_result_sourcedid, score)
```

Comment:
Create a service object in app/services  for this

Reply:
created ```app/services/lti_support.rb```